src/proto/mgmt/storage_query.proto

```diff
@@ -115,6 +115,18 @@ message DevReplaceResp {
 	string dev_state = 3; // BIO device state
 }
 
+message DevIdentifyReq {
+	string dev_uuid = 1; // UUID of VMD uuid
+	string dev_traddr = 2; // transport ID of the VMD SSD
+}
+
+message DevIdentifyResp {
+	int32 status = 1; // DAOS error code
+	string dev_uuid = 2; // UUID of VMD uuid
+	string dev_traddr = 3; // transport ID of the VMD SSD
+	string led_state = 4; // VMD LED state
```

Comment:
Not quite sure why it need to response uuid, traddr and led_state, it looks to me response a status is enough. Not a big deal though.